src/main/java/seedu/address/logic/parser/AlfredParser.java

```diff
@@ -96,7 +79,7 @@ public Command parseCommand(String userInput) throws ParseException {
 
         case DeleteCommand.COMMAND_WORD:
             logger.info("Deleting an existing Participant...");
```

Comment:
I think I was the one who added it, maybe for standardization I'll add logger information under all cases next time. 😄